source/fundamentals/authentication/mechanisms.txt

```diff
@@ -138,8 +138,7 @@ user.
 To connect to a MongoDB instance with ``MONGODB-AWS`` authentication
 enabled, specify the ``MONGODB-AWS`` authentication mechanism and pass
 your ``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY`` credentials to the
-driver when you attempt to connect. If your AWS login requires a session
-token, you must include your ``AWS_SESSION_TOKEN`` as well.
+driver when you attempt to connect. 
```

Comment:
If they are available in the environment, they don't have to be specified. This connection uri works for me:
```js
const uri = "mongodb+srv://jira-sync.pw0q4.mongodb.net/testDB?authSource=%24external&authMechanism=MONGODB-AWS&retryWrites=true&w=majority";
```

Let's expose the behavior the spec defines a little more clearly, especially so that users running in ec2 or something else can more easily use environment variables.